### DISCUSSION TITLE
Systemd startup package option

### DISCUSSION THREAD

**Discussion by rhodes-b** [ghostty-org/ghostty#3860] [Feature Requests, Ideas] [answered]
This sounds like a desirable feature like the desktop file that is bundled.

This sounds like a very nice way to package a headless ghostty for gtk-single-instance users which seems like a pretty common issue for people with tiling wm etc

looks like quite a bit of the work can just be taken from jcollie
https://github.com/jcollie/ghostty/commit/917a063cc3e5956f614eb60ce3ca3733020dfb85

**Comment by jcollie** [comment, marked as answer]
See #2442

**Reply by rhodes-b** [reply]
I was assuming this thread didnt exist per our discussion on discord will close
